gradle/libs.versions.toml

```diff
@@ -0,0 +1,105 @@
+[versions]
+slf4j = { strictly = "[2.0.7, 2.1[", prefer = "2.0.7" }
+slf4j-format = "0.5.1"
+groovy = "4.0.11"
+junit = "4.13.1"
+junit5 = "5.8.2"
+logback = "1.4.11"
+antlr = "4.13.1"
+hamcrest = "2.2"
+nekohtml = "1.9.22"
+htmlunit = "3.4.0"
+lang3 = "3.11"
+io = "2.11.0"
+codec = "1.13"
+unbescape = "1.1.6.RELEASE"
+xerces = "2.12.2"
+jstyleparser = "4.1.3"
+cssbox4 = "4.23"
+swingbox = "1.2.5"
+jsoup = "1.15.3"
+httpclient = "4.5.14"
+icu4j = { require = "[70,73.2[", prefer = "72.1" }
+stax2api = "4.2.1"
+woodstox = "6.5.0"
+languagetool = "6.1"
+lucene = "5.5.5"
+checkstyle = "8.45.1"
+guava = "31.0.1-jre"
+annotations = "23.0.0"
+jgit = "6.6.0.202305301015-r"
+svnkit = "1.10.11"
+ivy = "2.5.1"
+jackson = "2.14.2"
```

Comment:
Jackson has SnakeYAML dependency upgrade to Snakeyaml 2.0 from 1.33, to resolve CVE-2022-1471 in jackson 2.15.
We use YAML in tipoftheday module, so it is worth upgrade to jackson 2.15.2